man/bundle-config.ronn

```diff
@@ -84,8 +84,10 @@ The following is a list of all configuration keys and their purpose. You can
 learn more about their operation in [bundle install(1)][bundle-install].
 
 * `path` (`BUNDLE_PATH`):
-  The location on disk to install gems. Defaults to `$GEM_HOME` in development
-  and `vendor/bundle` when `--deployment` is used
+  The location on disk to install and locate gems. Defaults to `$GEM_HOME` in
+  development and `vendor/bundle` when `--deployment` is used. When the `path`
+  option is set, bundler will ignore `$GEM_HOME` and `$GEM_PATH` entirely and
+  require all gems be found in this location.
```

Comment:
How about "The location on disk where all gems in your bundle will be located. Bundle gems not found in this location will be installed by `bundle install`. Defaults to `Gem.dir`. When `--deployment` is used, defaults to `vendor/bundle`."